src/e2e/pages/landingPage.js

```diff
@@ -25,6 +28,18 @@ export class LandingPage {
     this.falseBannerCloseButton = page.locator('#close-button')
     this.scanEmailAddressInput = page.locator('#scan-email-address')
     this.checkForBreachesButton =  page.getByRole('button', {name: 'Check for breaches'})
+
+    // hero section
+    this.heroHeader = page.locator('.hero div h1').innerText()
+    this.heroDescription = page.locator('.hero p').innerText()
```

Comment:
Don't page objects generally just contain the selectors and possibly methods to interact with the page - but if tests want to view the contents of an element, they call `.innerText()` themselves?